pyneuroml/plot/PlotMorphologyVispy.py

```diff
@@ -1176,7 +1237,27 @@ def create_instanced_meshes(meshdata, plot_type, current_view, min_width):
             parent=current_view.scene,
         )
         # TODO: add a shading filter for light?
+        shaders.append(
+            ShadingFilter("smooth", shininess=1, ambient_light=(0.8, 0.8, 0.8, 0.6))
+        )
+        mesh.attach(shaders[counter])
         assert mesh is not None
+
+        def attach_headlight(current_view):
```

Comment:
now that you're saving the shaders, shouldn't all of this be done only once outside the for loop?

Reply:
Probably yes you are right